.lando.yml

```diff
@@ -85,7 +85,10 @@ tooling:
       - appserver: terminus backup:get portlandor.dev --element=database --to=/app/artifacts/database.tar.gz
       - database: echo "Importing database..."
       - database: cd /app && /helpers/sql-import.sh artifacts/database.tar.gz
-      - appserver: echo "Clearing cache..."
+      - appserver: echo "Truncating webforms and logs..."
+      - appserver: drush sql-query "truncate table webform_submission; truncate table webform_submission_data; truncate table webform_submission_log;"
```

Comment:
We could probably remove this watchdog truncation as well for the same reason as multidev.